addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -216,11 +216,13 @@ def test_mailing_contact_import(self):
         # Test that the context key "default_list_ids" is ignored (because we manually set list_ids)
         contact_import.with_context(default_list_ids=(first_list | second_list).ids).action_import()
 
+        first_list.invalidate_cache(['contact_ids'])
```

Comment:
the right usage of invalidate_cache is
```python
self.env['mailing.list'].invalidate_cache(['contact_ids'], ids=first_list)
```

(a lot of dev do this error, but if you do recordset.invalidate_cache() it will invalidate the entire model)

Reply:
@std-odoo 

If I try to invalidate only specific ids in the model then it doesn't work. Test cases start failing again. I don't know the perfect reason behind that. So, I have to invalidate the entire model currently.